doc/content/htgr/htr-pm/htr-pm_model.md

```diff
@@ -0,0 +1,492 @@
+# SAM Generic PBR Model
+
+*Contact: Zhiee Jhia Ooi, zooi@anl.gov, Stephen Bajorek, stephen.bajorek@nrc.gov*
+
+*Model link: [HTR-PM SAM Model](https://github.com/idaholab/virtual_test_bed/tree/main/htgr/htr-pm)*
+
+!tag name=SAM HTR-PM Model
+  image=https://mooseframework.inl.gov/virtual_test_bed/media/htrpm_sam/loop-schematics-VTB.png
+  description=This is a SAM HTR-PM reference plant model with steady-state and transient PLOFC simulations
+  pairs=reactor_type:HTGR
+                       reactor:HTR_PM
+                       geometry:Core_and_primary_loop
```

Comment:
dont make new categories here, you ll want to match one of these
https://mooseframework.inl.gov/virtual_test_bed/resources/filter/index.html

Reply:
changed to Core;Primary_Loop